### ISSUE TITLE
Failed to parse source map from

### ISSUE THREAD

**Issue by lasharela** [lob/lob-typescript-sdk#268]
## Summary

While running react application we are getting Wornings in console:

WARNING in ../../node_modules/@lob/lob-typescript-sdk/dist/index.mjs
Module Warning (from ../../node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '~/Development/dsl-web/dsl-apply-flow/node_modules/@lob/models/zip-editable.ts' file: Error: ENOENT: no such file or directory, open '~/Development/dsl-web/dsl-apply-flow/node_modules/@lob/models/zip-editable.ts'

## Expected Behavior

clean console log, without warnings. Because application is re-rendering multiple times, while initialization we see this warning many times.

**Comment by BennyKitchell** [comment]
@lasharela Would you mind trying to add `GENERATE_SOURCEMAP=false` to your .env and see if this unblocks you?

**Comment by lasharela** [comment]
@BennyKitchell  - Thanks. this works

**Comment by lasharela** [comment]
Unfortunately this is disabling sourcing in whole application. so, we can't go with this.

**Comment by BennyKitchell** [comment]
@lasharela  This is a known issue with webpack, I have been unable to find a workaround. 
For now, could you try adding this to your `webpack.config.js` file? 

This should block only source map complaints from lob. In the meantime, I will leave this issue open until I can find a worthy solution. 

```
const { merge } = require('webpack-merge');

module.exports = (config, context) => {
  return merge(config, {
    module: {
      rules: [
        {
          enforce: 'pre',
          test: /\.js$/,
          use: ['source-map-loader'],
          exclude:  ['/node_modules/@ lob']
        }
      ]
    }
  });
};
```

**Comment by jigmeloday** [comment]
hi, 
even i am having same issue
